model_analyzer/record/metrics_manager.py

```diff
@@ -589,9 +611,10 @@ def _run_perf_analyzer(
             max_retries=self._config.perf_analyzer_max_auto_adjusts,
             timeout=self._config.perf_analyzer_timeout,
             max_cpu_util=self._config.perf_analyzer_cpu_util,
+            model_type=self._config.model_type,
         )
 
-        metrics_to_gather = self._perf_metrics + self._gpu_metrics
+        metrics_to_gather = self._perf_metrics + self._llm_metrics + self._gpu_metrics
```

Comment:
will this always try to collect llm_metrics even when the model type isn't LLM?

Reply:
In this line, it will always try to add them, but if the model type is not LLM then `self._llm_metrics` will be blank.